consensus/phases/firstphase.go

```diff
@@ -111,11 +110,12 @@ func (fp *FirstPhaseImpl) Execute(ctx context.Context, pulse *insolar.Pulse) (*F
 	span.AddAttributes(trace.Int64Attribute("pulse", int64(pulse.PulseNumber)))
 	defer span.End()
 
-	var unsyncList network.UnsyncList
+	state := NewConsensusState(fp.NodeKeeper.GetSnapshotCopy())
 
 	pulseHash, pulseProof, err := fp.Calculator.GetPulseProof(entry)
-	if !fp.NodeKeeper.GetConsensusInfo().IsJoiner() {
-		unsyncList = fp.NodeKeeper.GetUnsyncList()
+	consensusInfo := fp.NodeKeeper.GetConsensusInfo()
```

Comment:
`ConsensusInfo` is a workaround structure that contains additional info about consensus and it's lifetime is longer than consensus lifetime, so no, it can't be a part of `ConsensusState`. The main idea is that we will remove `ConsensusInfo` in future after another round of refactoring.

Reply:
Yeah, but `ConsensusInfo` is actual only for *current* consensus, isn't it?
I mean it could be called once and used across all phases.

Reply:
Added `ConsensusInfo` to `ConsensusState`